components/hab/src/command/pkg/binlink.rs

```diff
@@ -142,104 +240,6 @@ fn is_dest_on_path(dest_dir: &Path) -> bool {
     }
 }
 
-struct Binlink {
```

Comment:
Because I found it confusing that it was defined at the bottom of the file.  Typically our structs and such are defined before they're used, which I get is not a requirement.  I do find it makes the file more readable though.